[2024-01-17T14:30:55.519-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_coleta_teste.saopaulo scheduled__2024-01-17T17:29:02.840471+00:00 [queued]>
[2024-01-17T14:30:55.522-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_coleta_teste.saopaulo scheduled__2024-01-17T17:29:02.840471+00:00 [queued]>
[2024-01-17T14:30:55.522-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T14:30:55.528-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): saopaulo> on 2024-01-17 17:29:02.840471+00:00
[2024-01-17T14:30:55.532-0300] {standard_task_runner.py:57} INFO - Started process 15244 to run task
[2024-01-17T14:30:55.534-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_coleta_teste', 'saopaulo', 'scheduled__2024-01-17T17:29:02.840471+00:00', '--job-id', '1327', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp9sq0ai9d']
[2024-01-17T14:30:55.535-0300] {standard_task_runner.py:85} INFO - Job 1327: Subtask saopaulo
[2024-01-17T14:30:55.555-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_coleta_teste.saopaulo scheduled__2024-01-17T17:29:02.840471+00:00 [running]> on host desktop
[2024-01-17T14:30:55.579-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_coleta_teste' AIRFLOW_CTX_TASK_ID='saopaulo' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T17:29:02.840471+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T17:29:02.840471+00:00'
[2024-01-17T14:30:56.745-0300] {logging_mixin.py:154} INFO - true
[2024-01-17T14:30:56.745-0300] {logging_mixin.py:154} INFO - <Response [200]>
[2024-01-17T14:30:57.519-0300] {logging_mixin.py:154} INFO - Called | 1705512657.513592 | sp
[2024-01-17T14:30:57.520-0300] {logging_mixin.py:154} INFO - sp - Error on save as CSV 1705512657.513592: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet'.
A suitable version of pyarrow or fastparquet is required for parquet support.
Trying to import the above resulted in these errors:
 - Missing optional dependency 'pyarrow'. pyarrow is required for parquet support. Use pip or conda to install pyarrow.
 - Missing optional dependency 'fastparquet'. fastparquet is required for parquet support. Use pip or conda to install fastparquet.
[2024-01-17T14:30:57.521-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T14:30:57.524-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_coleta_teste, task_id=saopaulo, execution_date=20240117T172902, start_date=20240117T173055, end_date=20240117T173057
[2024-01-17T14:30:57.589-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T14:30:57.599-0300] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
